string topic_name 

---
int64 result